src/bls12_381/ec.rs

```diff
@@ -1419,6 +1481,24 @@ pub mod g2 {
             }
         }
 
+        fn get_c1() -> Fq2 {
```

Comment:
Right, so I changed it into `SWENC_CONST[01]` as in `Shallue–van de Woestijne constant`